src/views/Predictions/Positions.tsx

```diff
@@ -22,19 +23,23 @@ const StyledSwiper = styled.div`
     width: 320px;
   }
 `
-const Positions: React.FC = () => {
+const Positions: React.FC<{ view?: string }> = ({ view }) => {
   const { setSwiper } = useSwiper()
   const rounds = useGetSortedRounds()
-  const initialIndex = Math.floor(rounds.length / 2)
+  const currentEpoch = useGetCurrentEpoch()
+  const liveEpoch = currentEpoch > 0 ? currentEpoch - 1 : currentEpoch
```

Comment:
When `useGetCurrentEpoch` returns 0, the live epoch isn't -1, so added some logic to catch and return 0. This may be unnecessary given this scenario may only be true in early testing?